src/common/VisualViewportCSSVariables.tsx

```diff
@@ -13,28 +13,31 @@ const isIOS = (): boolean =>
 
 /**
  * Maintains CSS variables for the visual viewport width and height.
- * Use with fallbacks, e.g. var(--ios-vvh, 100vh)
+ * Use with fallbacks, e.g. var(--vvh, 100vh)
  * https://developer.mozilla.org/en-US/docs/Web/API/Visual_Viewport_API
- *
- * This is important for iOS where we want to avoid the keyboard being
- * drawn over the toolbar at the bottom.
- *
- * We skip it on other platforms and rely on the fallback.
  */
 const VisualViewPortCSSVariables = () => {
+  const addStyleInnerText = (innerText: string) => {
+    let style = document.getElementById(styleId);
+    if (!style) {
+      style = document.head.appendChild(document.createElement("style"));
+      style.id = styleId;
+    }
+    style.innerText = innerText;
+  };
+
   useEffect(() => {
-    // If we remove the iOS check then we should look carefully at resize performance.
-    if (!window.visualViewport || !isIOS()) {
+    // svh on Safari in iOS does not resize when virtual keyboard is present
+    if (CSS.supports("height: 100svh") && !isIOS) {
+      addStyleInnerText(":root { --vvh: 100svh }");
+      return;
+    }
+    if (!window.visualViewport) {
       return;
     }
```

Comment:
Previously, we only used visualViewport when it is iOS. Was not sure why we limited to only iOS in the first place, since `window.visualViewport` appears to work well in Android as well 🤔 I used it as a fallback option for Android as well